runtime/vm/jvminit.c

```diff
@@ -2547,9 +2547,7 @@ modifyDllLoadTable(J9JavaVM * vm, J9Pool* loadTable, J9VMInitArgs* j9vm_args)
 			jvmti = TRUE;
 		} else
 #if defined(J9VM_OPT_SIDECAR)
-		if ((J2SE_VERSION(vm) & J2SE_VERSION_MASK) >= J2SE_16) {
-			jvmti = TRUE;
-		}
+		jvmti = TRUE;
```

Comment:
Note: this code can be further simplified if `J9VM_OPT_SIDECAR`is always enabled which is the case for all current specs but leave that to another PR instead.

Reply:
Is there some reason not to just delete all this code and replace it with `jvmti = TRUE;`? Seems an easy change.

Reply:
As per previous note, I preferred to leave further simplification to another PR for the flag `J9VM_OPT_SIDECAR`.
However just noticed that original code already relies on `J9VM_OPT_SIDECAR` is on, other syntax error occurs. 

```
#if defined(J9VM_OPT_JVMTI)
		if ((strstr(testString, VMOPT_AGENTLIB_COLON)==testString) || (strstr(testString, VMOPT_AGENTPATH_COLON)==testString)) {
			ADD_FLAG_AT_INDEX( ARG_REQUIRES_LIBRARY, i );
			jvmti = TRUE;
		} else   ---> following code won't exist if J9VM_OPT_SIDECAR is not enabled.
#if defined(J9VM_OPT_SIDECAR)
		if ((J2SE_VERSION(vm) & J2SE_VERSION_MASK) >= J2SE_16) {
			jvmti = TRUE;
		}
#endif
#endif
```

Will remove `jvmti` completely instead.